x/concentrated-liquidity/math/math.go

```diff
@@ -77,14 +75,14 @@ func CalcAmount0Delta(liq, sqrtPriceA, sqrtPriceB sdk.Dec, roundUp bool) sdk.Dec
 		// - adding liquidity (request user to provide more tokens in in favor of the pool)
 		// The denominator is truncated to get a higher final amount.
 		denom := sqrtPriceA.MulTruncate(sqrtPriceB)
-		return liq.Mul(diff).Quo(denom).Ceil()
+		return liq.Mul(diff).QuoMut(denom).Ceil()
 	}
 	// These are truncated at precision end to round in favor of the pool when:
 	// - calculating amount out during swap
 	// - withdrawing liquidity
 	// The denominator is rounded up to get a smaller final amount.
 	denom := sqrtPriceA.MulRoundUp(sqrtPriceB)
-	return liq.MulTruncate(diff).QuoTruncate(denom)
```

Comment:
this is so sad that we have to manually do this

Reply:
yeah lol